src/functionalTest/java/uk/gov/hmcts/reform/pip/account/management/SubscriptionTest.java

```diff
@@ -53,19 +53,24 @@ class SubscriptionTest extends AccountHelperBase {
 
     private static final String LOCATION_ID = randomLocationId();
     private static final String LOCATION_NAME = "TestLocation" + LOCATION_ID;
-    private static final String USER_ID = UUID.randomUUID().toString();
     private static final String USER_ID_HEADER = "x-user-id";
     private static final String BEARER = "Bearer ";
     private static final String LIST_LANGUAGE = "ENGLISH";
     private static final String LIST_LANGUAGE_CONFIGURE = "WELSH";
     private static final ListType LIST_TYPE = ListType.CIVIL_DAILY_CAUSE_LIST;
 
     private PiUser systemAdminAccount;
+    private UUID verifiedUserId = UUID.randomUUID();
 
     @BeforeAll
     public void setup() throws JsonProcessingException {
         bearer = Map.of(AUTHORIZATION, BEARER + accessToken);
         systemAdminAccount = createSystemAdminAccount();
+        String createdUserId = getCreatedAccountUserId(
+            createAccount("test-verfied-user@test.com", UUID.randomUUID().toString())
```

Comment:
We should use the generateEmail function here rather than a fixed one, to ensure it gets deleted after each run

Reply:
done